editors/code/src/client.ts

```diff
@@ -1,9 +1,14 @@
-import * as lc from 'vscode-languageclient';
+import * as lc from 'vscode-languageclient/node';
 import * as vscode from 'vscode';
 
 import { Config } from './config';
-import { CallHierarchyFeature } from 'vscode-languageclient/lib/callHierarchy.proposed';
-import { SemanticTokensFeature, DocumentSemanticsTokensSignature } from 'vscode-languageclient/lib/semanticTokens.proposed';
+//import { CallHierarchyFeature } from 'vscode-languageclient/lib/callHierarchy.proposed';
```

Comment:
These could be removed, right?

Reply:
These dependencies have been replaced with packages/versions that don't end with ".proposed"

Reply:
Yep I mean the comments could be removed.

Reply:
Yes, I tend comment out lines if I'm not sure if I'm going to need them again or not.